coordinator/src/main/scala/filodb.coordinator/DatasetCoordinatorActor.scala

```diff
@@ -168,6 +168,7 @@ private[filodb] class DatasetCoordinatorActor[K](projection: RichProjection[K],
       logger.warn("This should not happen. Not starting flush; $curReprojection; $flushingTable")
       return
     }
```

Comment:
The raw file doesn't show 'return' in the signature, but above it 'looks' as though you've got a 'return' for a Unit signature.

     private def reportStats(): Unit = {
        logger.info(s"MemTable active table rows: $activeRows")
        logger.info(s"MemTable flushing rows: $flushingRows")
    }